### ISSUE TITLE
Change password dialog continues without you asking it to

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#5329]
### Description

The dialog appears, but then the page reloads and you're left wondering what the dialog said.

This is not acceptable - it should at least let me export my e2e keys safely before destroying them.

### Steps to reproduce

- Change your password

### Version information

- **Platform**: web (in-browser)
- **Browser**: Chrome 61
- **OS**: Windows 10
- **URL**: riot.im/develop

**Comment by uhoreg** [comment]
related to https://github.com/vector-im/riot-web/issues/2671

**Comment by turt2live** [comment]
Just realized that 0.13 got released after this was labeled as a release blocker :(

**Comment by turt2live** [comment]
@lampholder @ara4n  In light of a couple people (and me) running into this, can it be squeezed into 0.14? It is afterall a release-blocker 😇

**Comment by turt2live** [comment]
@lampholder This is another desperate request to have this fixed.

**Comment by lampholder** [comment]
Hey! Sorry for ignoring your first desperate plea (@-mentions now reach me as an email). Is it still broken? I just went to reset my password on a test account and however I submitted (pressing <kbd>Enter</kbd> or clicking the `CHANGE PASSWORD` button) I still got the confirmation dialogue which stayed in place until I took an action:

![image](https://user-images.githubusercontent.com/1922197/37338703-2551e5b0-26b0-11e8-9ca3-807dbe190576.png)
